m4/geany-docutils.m4

```diff
@@ -70,3 +70,16 @@ AC_DEFUN([GEANY_CHECK_DOCUTILS_PDF],
 	AM_CONDITIONAL([WITH_RST2PDF], [test "x$geany_enable_pdf_docs" != "xno"])
 	GEANY_STATUS_ADD([Build PDF documentation], [$geany_enable_pdf_docs])
 ])
+
+dnl
+dnl GEANY_CHECK_PYTHON
+dnl For gtkdoc header generation
+dnl
+AC_DEFUN([GEANY_CHECK_PYTHON],
+[
+	AM_PATH_PYTHON([2.7], [], [])
```

Comment:
You must be kidding me right? Python 2.7 is over 5 years old. python 3 is even older (nearly 8 years). "Recent" is not quite right. Even depending on py3 would be perfectly reasonable but I still made the effort to make it 2.7 compatible.

Anyway, I dont have something older than  2.7 so I can't actually tell. Might work with  older versions.